lib/widgets/courseList.dart

```diff
@@ -40,7 +40,7 @@ class CourseListState extends State<CourseList> {
   final TextEditingController c1 = new TextEditingController();
   final bool shouldFilterByFavorites;
   bool shouldSearch = false;
-  String filter = "09";
+  String filter = "13";
   String searchValue = "";
```

Comment:
Ist das richtig dass du standardmäßig nach fk13 suchen willst. Wir erhalten doch nur von fk09 sachen

Reply:
Nach Screenshot erhalten wir mittlerweile auch andere departments oder?

Reply:
That's the reason, why this pull request exists ;-)

Reply:
On nine CiE is placed under department 13, for this reason I though it might be good to use it as a default.